scapy_ssl_tls/ssl_tls.py

```diff
@@ -923,7 +931,7 @@ class TLSCertificateType(PacketNoPayload):
 class TLSCADistinguishedName(PacketNoPayload):
     name = "TLS CA Distinguished Name"
     fields_desc = [XFieldLenField("length", None, length_of="dn", fmt="H"),
-                   PacketLenField("ca_dn", None, x509.X509v3Ext, length_from=lambda x:x.length)]
+                   PacketLenField("ca_dn", None, x509.X509_Extension, length_from=lambda x:x.length)]
```

Comment:
I'm not entirely sure if ``X509_Extension`` is the right replacement for the missing ``X509v3Ext``.

``X509v3Ext`` must have existed at some point, so I'll check the history of the repo for renames and/or refactors.